### PR TITLE
:3

### PR DESCRIPTION
**Pull request by obstructive**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        ]
+        for future in concurrent.futures.as_completed(futures):
+            try:
+                future.result()
+            except requests.exceptions.HTTPError as error:
+                print(f'An error occurred: {error}')
+
+
+if __name__ == '__main__':
+    main()
```
Comment (hydromorphine) [inline main.py:83]: thug

Comment (obstructive) [reply main.py:83]: thugged

**Review by obstructive — COMMENTED** [review]
fire code ngl @1x6 merge it

File: main.py (hunk 1 above)
Comment (ba-g) [reply main.py:83]: dick

**Comment by 1x6** [comment]
gpt ass code

**Comment by obstructive** [comment]
> gpt ass code

docs were written by gpt, everything else by me and my 3 brainlets in a discord vc at night

**Pull request merged**